sema4ai/package.json

```diff
@@ -1,17 +1,23 @@
 {
     "name": "sema4ai",
-    "displayName": "Sema4.ai",
```

Comment:
Sorry, this isn't as easy here... you should change `sema4ai/codegen/codegen_package.py` and then run it to update this file...

Reply:
Codegen updated and executed